OpenRA.Mods.Common/Traits/BotModules/UnitBuilderBotModule.cs

```diff
@@ -72,7 +78,8 @@ void IBotNotifyIdleBaseUnits.UpdatedIdleBaseUnits(List<Actor> idleUnits)
 
 		void IBotTick.BotTick(IBot bot)
 		{
-			if (requestPause.Any(rp => rp.PauseUnitProduction))
+			// There is no point that produces tons of the items when the cash running low
```

Comment:
```suggestion
			// There is no point in producing tons of items when the cash running low
```
Your spelling is so bad, I have a hard time understanding it. Please use @languagetool-org or similar to check grammar before submitting.

Reply:
sorry and I am practicing coding without the auto-correct tool recently, in addition, thanks for the link.

Ummm and this looks strange:
![umm](https://user-images.githubusercontent.com/13763394/182740349-f36bd54b-254e-4973-b904-fd578ebb38a7.PNG)

Reply:
Maybe I need a clearer expression on these.